static/js/containers/ProductDetailEnrollApp.js

```diff
@@ -63,39 +173,45 @@ export class ProductDetailEnrollApp extends React.Component<Props> {
                 Enroll now
               </a>
             ) : run && isWithinEnrollmentPeriod(run) ? (
-              <Fragment>
-                <form action="/enrollments/" method="post">
-                  <input
-                    type="hidden"
-                    name="csrfmiddlewaretoken"
-                    value={csrfToken}
-                  />
-                  <input type="hidden" name="run" value={run ? run.id : ""} />
-                  <button
-                    type="submit"
-                    className="btn btn-primary btn-gradient-red highlight"
-                  >
-                    Enroll now
-                  </button>
-                </form>
-              </Fragment>
+              <button
```

Comment:
We don't want this new UI available yet, so could you:

- Add a feature flag for ecommerce and pass it down in `SETTINGS`
- Use the current UI if this flag isn't set to `True`